.github/workflows/cron-daily-fuzz.yml

```diff
@@ -81,8 +86,6 @@ jobs:
     if: ${{ !github.event.act }}
     needs: fuzz
     runs-on: ubuntu-24.04
-    permissions:
-      contents: read
```

Comment:
I think we want this still, right. I'm not a github actions guy but this was added recently by @storopoli and he seems to know his way around github actions.

Reply:
I'll adjust the generate-files.sh so that it can reproduce the files with only the minimum required changes.